README.md

```diff
@@ -207,37 +242,62 @@ Asker.bodyEncoders.trimText = function(body, setContentType) {
 
 ### The problem
 
-At this point of time (version `0.10` and below) node.js provides a socket pool manager that works as follows:
+For Node.js versions 0.10 and below an internal socket pool manager works as follows:
 
 * by default `globalAgent` is used for all outgoing http requests;
 * each `Agent` instance, including `globalAgent`, has a `maxSockets` property, which you can change;
-* socket limit is set for each unique __host-port pair__, that is served by this particular `Agent`.
+* socket limit is set for each unique **host:port pair**, that is served by this particular `Agent`.
 
-That is sometimes an unwanted behaviour. Let's take an example.
+The pool may behaves unexpectedly in some cases. For example, you have two backend servers: `backend:3000` and `backend:4000`.
+First backend is indispensable for the application, but the second one is complementary (e.g. it makes an http call for advertisements you show later).
 
-You have two backends: `backend:3000` and `backend:4000`. First backend is indispensable for the application, but the second one is complementary. E.g., it makes http calls for the advertisements that you show later.
+Under the heavy load this additional backend (which is usually less fault-tolerant) may occupy all sockets that OS provides
+for the whole Node.js process, because default pool manager cares only about **host:port pairs** and `defaultAgent`
+does not correct each backend's socket limit according to process limit.
 
-Under heavy load this additional backend (which is usually less fault-tolerant) may occupy all sockets that OS provides for the whole `node` process, because default pool manager cares only about __host-port__ pairs. `defaultAgent` does not anyhow correct each backend' socket limit according to process limit.
+In the scenario above both backends will be unavailable for us, even if first indispensable server works fine.
 
 ### Solution
 
-How `Asker` can help you manage this problem? `Asker` can create a custom instance of `http.Agent` for any given backend. And you can set up a `maxSockets` property by calculating each backend priority.
+How asker can help you manage this problem? You can create a custom instance of `http.Agent` for any given backend.
+And you can set a `maxSockets` property by calculating each backend priority. Asker gives you an API to help with that.
```

Comment:
https://www.google.ru/?gfe_rd=cr&ei=QjRBVuG9MpTANOz8segO&gws_rd=ssl#newwindow=1&safe=off&q=gives+you+API